Dockerfile.aarch64

```diff
@@ -0,0 +1,29 @@
+FROM frozeneye/aarch64-nodejs:latest
```

Comment:
That may be a better option to derive from aarch64 (providing that account is also known). The concern about mounting the socket is also valid but not specific to this pull request.